docs/src/app/components/pages/components/DatePicker/ExampleInternational.jsx

```diff
@@ -4,26 +4,43 @@ import areIntlLocalesSupported from 'intl-locales-supported';
 
 let DateTimeFormat;
 
-// Use the native Intl if available
+/**
+ * Use the native Intl.DateTimeFormat if available, or a polyfill if not.
+ */
 if (areIntlLocalesSupported('fr')) {
```

Comment:
Well, that's not a big deal. I'm pretty sure that if 'fr' is available, 'en-US' is also available.